File: tjhsst comp sci 2/Java/Units/Unit1/Lab00.java
//Vivian Feng and Shriya Muthukumar
//3.9.2020
import edu.fcps.karel2.Display;
import edu.fcps.karel2.Robot;

public class Lab00
{
    public static void main(String[] args)
    {
        Display.openWorld("maps/first.map");
        Display.setSize(10,10);
        Display.setSpeed(10);

        Robot karel = new Robot();

        karel.move();
        karel.pickBeeper();
        karel.move();
        karel.turnLeft();
        karel.move();
        karel.putBeeper();
        karel.move();
        karel.turnLeft();
        karel.turnLeft();
    }
}

File: tjhsst comp sci 2/pyKarel Student Code/lab00.py
#Name: Vivian
#Date: 8/26/2019

from pyKarel import *

wld = World("first")
karel = Robot(wld)

karel.move()
karel.pickBeeper()
karel.move()
karel.turnLeft()
karel.move()
karel.putBeeper()
karel.move()
karel.turnLeft()
karel.turnLeft()
wld.mainloop()

